pkg/crd/templates/testtrigger.tmpl

```diff
@@ -103,6 +103,48 @@ spec:
   {{- if .Action }}
   action: {{ .Action }}
   {{- end }}
+  {{- if .ActionParameters }}
+  actionParameters:
+    {{- if ne (len .ActionParameters.Config) 0 }}
+    config:
+      {{- range $key, $value := .ActionParametersConfig }}
+      {{ $key }}: {{ $value }}
```

Comment:
Will it work, given that there may be any data deep down in the `$value`?

Reply:
This is just `Config map[string]string `json:"config,omitempty"`` right?